conan/tools/cmake/toolchain.py

```diff
@@ -225,6 +225,20 @@ def context(self):
         return {"cppstd": cppstd, "cppstd_extensions": cppstd_extensions}
 
 
+class FindPackageDirVarsBlock(Block):
+    template = textwrap.dedent("""
+        # Help locating the find_package config files
+        {% for name in find_names %}
+        set({{ name }}_DIR "${CMAKE_BINARY_DIR}")
```

Comment:
Why ``CMAKE_BINARY_DIR``? Shouldn't be the ``generators_folder``?

Reply:
Probably `${CMAKE_CURRENT_LIST_DIR}` to be consistent with `cmake_module_path`